test/modules/packages/Crypto/saru/blowfish/bfTest1.chpl

```diff
@@ -0,0 +1,19 @@
+proc main() {
+  use Crypto;
+
+  var bf = new Blowfish("cbc");
+
+  var msg = new CryptoBuffer("hello world");
+
+  // static values assumed for testing purposes
+  var iv = new CryptoBuffer("iv");
+  var key = new CryptoBuffer("key");
+
+  writeln("MSG: ", msg.toHex());
+
+  var ct = bf.encrypt(msg, key, iv);
+  writeln("CT: ", ct.toHex());
+
+  var pt = bf.decrypt(ct, key, iv);
+  writeln("PT: ", pt.toHex());
```

Comment:
Please avoid leaking your classes by calling delete on all allocated classes like in #7559.  You can check for leaks by compiling the tests and running them with the `--memLeaks` flag.

```
chpl bfTest1.chpl
bfTest1 --memLeaks
```